pkg/operations/horizontal_scaling.go

```diff
@@ -433,3 +474,72 @@ func (hs horizontalScalingOpsHandler) getCompExpectedOfflineInstances(
 	}
 	return compOfflineInstances
 }
+
+// validate if there is any instance specified in the request that is not exist, return error.
+// if ignoreStrictHorizontalValidation is empty, it would validate the instances if they are already offlined or onlined.
```

Comment:
Update the comment since ignoreStrictHorizontalValidation does not occur in the code.